.github/workflows/e2e-android.yml

```diff
@@ -0,0 +1,69 @@
+name: E2E - Android
+on:
+  workflow_call:
+    inputs:
+      android-api-level:
+        required: true
+        type: number
+
+jobs:
+  android:
+    env:
+      BASH_ENV: ~/.profile
+    name: Android (SDK ${{ inputs.android-api-level }})
+    runs-on: android-e2e-group
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+          check-latest: true
+      - name: Install package dependencies
+        run: yarn || yarn --network-concurrency 1
+      - name: Fail if someone forgot to commit "yarn.lock"
+        run: git diff --exit-code
+      - name: Check E2E wallet balance
+        run: NODE_OPTIONS='--unhandled-rejections=strict' yarn ts-node ./e2e/scripts/check-e2e-wallet-balance.ts
+      - name: Copy Emulator Config
+        run: mkdir -p ~/.android/avd/Pixel_API_${{ inputs.android-api-level }}_AOSP_x86_64.avd/ && cp e2e/conf/ci_avd_conf.ini ~/.android/avd/Pixel_API_${{ inputs.android-api-level }}_AOSP_x86_64.avd/config.ini
```

Comment:
Why is this step necessary?

Reply:
This sets up the ini config with some specific values for the emulators. We could move this over to the infrastructure repo to avoid running it on every PR.

Reply:
We'll need to adjust the ci_avd_conf.ini or create slightly different one for each device as there are some emulator specific values such as:
```
image.sysdir.1=system-images/android-29/default/x86_64/
sdcard.path=/Users/distiller/.android/avd/Pixel_API_29_AOSP_x86_64.avd/sdcard.img
```

As I see it can either create multiple ci_avd_conf.ini files or have the setup in infrastructure repo.

Reply:
We could also edit the `ci_avd.conf.ini` to exclude these items and see if that will work.

Reply:
You can see the `.ini` file we have committed, [here](https://github.com/valora-inc/wallet/blob/2b28b691640c52bdeeb1b49a7751c5bdf3b764fe/e2e/conf/ci_avd_conf.ini).  This step is just copying that file into the AVD directory, so it's used when the emulator is spun up.

Reply:
Turns out this wasn't the right fix. The `.ini` file that's being copied here may be ignored, as the snapshot is based off a different `.ini` file.

To be sure, we're going to do the `.ini` copying in the provisioning step instead.